XenAdmin/Wizards/ImportWizard/ImageVMConfigPage.cs

```diff
@@ -81,6 +81,7 @@ public override void PopulatePage()
 			m_upDownMemory.Value = m_upDownMemory.Minimum;
 			m_upDownCpuCount.Value = m_upDownCpuCount.Minimum;
 			m_upDownAddSpace.Value = m_upDownAddSpace.Minimum;
+			bootModesControl1.CheckBIOSBootMode();

```

Comment:
I notice that when I click Previous button and then click Next button, some controls will clear the users' input value.
So I would like the BootModeControl to keep consistent with them.